EIPS/eip-5749.md

```diff
@@ -64,17 +65,17 @@ The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL
 
 ```
 
-Type `EIP1193Provider` is well documented at [EIP-1193](./eip-1193.md)
+Type `EIP1193Provider` is documented at [EIP-1193](./eip-1193.md)
 
-```
+```text
 interface ProviderInfo
 uuid: UUIDv4 unique to the provider
 name: Name of the Wallet
 icon: base64 encoded svg image
 description: Description for your wallet
 ```
 
-```
+```text
 interface EVMProvidersType
 key is RECOMMENDED to be the name of the extension
 ```
```

Comment:
```suggestion
```typescript
/**
 * The type of `window.evmproviders`
 */
interface EVMProviders {
  /**
   * The key is RECOMMENDED to be the name of the extension in snake_case. It MUST contain only lowercase letters, numbers, and underscores.
   */
  [index: string]: string;
}
```
```